locales/es/es.json

```diff
@@ -272,20 +273,20 @@
     "If you did not receive the email": "Si no ha recibido el correo electrónico",
     "If you did not request a password reset, no further action is required.": "Si no ha solicitado el restablecimiento de contraseña, omita este mensaje de correo electrónico.",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Si no tiene una cuenta, puede crear una haciendo clic en el botón de abajo. Después de crear una cuenta, puede hacer clic en el botón de aceptación de la invitación en este correo electrónico para aceptar la invitación del equipo:",
-    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.",
+    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Si necesita agregar información de contacto específica o de impuestos a sus recibos, como su nombre comercial completo, número de identificación VAT o dirección de registro, puede agregarlo aquí.",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Si tiene problemas para hacer clic en el botón \":actionText\", copie y pegue la siguiente URL \nen su navegador web:",
     "Increase": "Incrementar",
     "India": "India",
     "Indonesia": "Indonesia",
     "Invalid signature.": "Firma no válida.",
-    "Iran, Islamic Republic of": "Iran, Islamic Republic of",
+    "Iran, Islamic Republic of": "Irán, República Islámica de",
```

Comment:
you have two translations for the same country ?

Reply:
@caouecs, i didn't write this duplicate keys in file `source/en.json` but you. I only translate. For example
[https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L282](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L282) Lines 281 and 282 are the same contry. The same goes for the other cases.
If you don't want repeated countries it's necesary modify `source/en.json` in several parts.
- [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L330](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L330) 330 and 331
- [https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) 553 and 554
- etc ...